core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -256,7 +319,24 @@ public void finished(long requestSize, long responseSize, long latencyMs) {
     /**
      * Indicate that a request has failed with an exception.
      */
-    public void exception() {
+    public void exception(final RequestEvent event) {
+      int errorCode = -1;
+      if (event.getException() instanceof RestException) {
+        errorCode = ((RestException) event.getException()).getErrorCode();
+      }
+      if (errorCode < 0) {
+        errorSensorUnknown.record();
+      } else if (errorCode < 200) {
+        errorSensor100s.record();
+      } else if (errorCode < 300) {
+        errorSensor200s.record();
+      } else if (errorCode < 400) {
+        errorSensor300s.record();
+      } else if (errorCode < 500) {
+        errorSensor400s.record();
+      } else {
```

Comment:
i can't imagine it'd work anyway, but `>=600` should be `errorSensorUnknown` as well (same as `<0`, which I also can't imagine wouldn't work).